### ISSUE TITLE
Unable to Change Value with Talkback Active (Android Screen reader)

### ISSUE THREAD

**Issue by barr1209** [tb/ng2-nouislider#245]
In Android mobile, the user should be able to swipe up and down to change the value of the slider using Talkback screen reader. However, when Talkback is active this does not function – swiping up or down does nothing and Talkback users cannot operate. 

**Steps to reproduce:**
1. Open the webpage in android device with Talkback active 
2. Swipe until a slider is focused 
3. Swipe up to attempt to change the value 
4. The value does not change 
5. Double-tap to attempt to activate the element 
6. Swipe up to attempt to change the value 
7. The value does not change
